packages/loader/src/gltf/GLTFResource.ts

```diff
@@ -0,0 +1,18 @@
+import { AnimationClip, BufferMesh, Camera, EngineObject, Entity, Material, Skin, Texture2D } from "@oasis-engine/core";
+import { IGLTF } from "./schema";
+
+export class GLTFResource extends EngineObject {
```

Comment:
Write comments for this class,this is a common class used by users and is very important.

Reply:
done